libs/fetcher.client.js

```diff
@@ -10,65 +10,14 @@
  * @module Fetcher
  */
 var REST = require('./util/http.client');
-var DEFAULT_GUID = 'g0';
+var defaultConstructGetUri = require('./util/defaultConstructGetUri');
```

Comment:
looks like this file can move to modern code too, for another PR

Reply:
Oh, only if we completely drop support for IE11 (or add ts or babel to transpile correctly).

Reply:
Yup, I created #344  to convert to TS